Applications/Games/The Sims 4/Local/script.js

```diff
@@ -0,0 +1,36 @@
+const LocalInstallerScript = include("engines.wine.quick_script.local_installer_script");
+const {LATEST_STAGING_VERSION} = include("engines.wine.engine.versions");
+const AppResource = include("utils.functions.apps.resources");
+const vcrun6sp6 = include("engines.wine.verbs.vcrun6sp6");
+const MFC42 = include("engines.wine.verbs.mfc42");
+const dotnet20sp2 = include("engines.wine.verbs.dotnet20sp2");
+const vcrun2010 = include("engines.wine.verbs.vcrun2010");
+const vcrun2013 = include("engines.wine.verbs.vcrun2013");
+const D9VK = include("engines.wine.verbs.d9vk");
+const regedit = include("engines.wine.plugins.regedit");
+const overrideDLL = include("engines.wine.plugins.override_dll");
```

Comment:
This is a good idea sadly we haven't change the plugins yet to return modules. So for plugins:

```javascript
include("engines.wine.plugins.regedit");
include("engines.wine.plugins.override_dll");
```

is still correct.